lib/type1.g

```diff
@@ -613,7 +613,8 @@ BIND_GLOBAL( "Objectify", function ( type, obj )
     elif IS_REC( obj )  then
         SET_TYPE_COMOBJ( obj, type );
     fi;
-    if not IsNoImmediateMethodsObject(obj) then
+    if not ( IGNORE_IMMEDIATE_METHODS
+             or IsNoImmediateMethodsObject(obj) ) then
```

Comment:
You need to run `make docomp` (once in a regular GAP build, and once in a build with `hpcgap` enabled) so that the files `src/c_type1.c` and `src/hpc/c_type1.c` are updated.

(This also work with out-of-tree builds, which is the most convenient way to update both the regular and the HPC version with a single clone)